users/views.py

```diff
@@ -1,3 +1,77 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+
+from django.conf import settings
+from django.http import JsonResponse
+from django.forms import ValidationError
+import bcrypt
+import jwt
+
+from cores.validations import validate_email, validate_password
+from .models import *
+
+
+def check_email(request):
+    try:
+        if not request.method == 'POST':
+            return JsonResponse({"message" : "INVALID_METHOD"}, status=405) 
+        data  = json.loads(request.body)
+        email = data['email']
+
+        validate_email(email)
+
+        IS_EXITS = User.objects.filter(email = email).exists()
+        return JsonResponse({'message' : IS_EXITS}, status = 200)
+
+    except KeyError:
+        return JsonResponse({'message' : 'KEY_ERROR'}, status = 400)
+
+def sign_up(request):
+        try:
+            if not request.method == 'POST':
+                return JsonResponse({"message" : "INVALID_METHOD"}, status=405) 
+            data       = json.loads(request.body)
+            email      = data['email']
+            password   = data['password']
+            last_name  = data['last_name']
+            first_name = data['first_name']
+
+            validate_password(password)
+
+            User.objects.create(
+                email      = email,
+                password   = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8'),
+                last_name  = last_name,
+                first_name = first_name
+            )
```

Comment:
return문과 User.objects.create() 함수 호출 사이에 한 줄 띄워주세요-!
```suggestion
            )
            
```